### PR TITLE
added script for daily operator balance monitoring

### PR DESCRIPTION
**Pull request by IvanAdamovNeon**

### CODE REVIEW

**Comment by IvanAdamovNeon** [comment]
operators.lson example
[operators.json](https://github.com/user-attachments/files/17300852/operators.json)

if Solana api is still down, we can read earlier save transactions from csv instead of pullinmg from Solana
[data_dump.csv](https://github.com/user-attachments/files/17300854/data_dump.csv)

report example
[operator_daily_balance.pdf](https://github.com/user-attachments/files/17300860/operator_daily_balance.pdf)

**Comment by IvanAdamovNeon** [comment]
@kristinaNikolaevaa Operator Neon deposit is fetched from Indexer Postgres neon_transactions.sum_gas_used

File: deploy/requirements/click.txt [hunk 1]
```diff
@@ -22,4 +22,8 @@ polling2==0.5.0
 psycopg2-binary==2.9.9
 sqlalchemy==2.0.31
 pandas==2.2.2
-matplotlib==3.9.1
```
Comment (gigimon) [inline deploy/requirements/click.txt:25]: I think we need additional requirement file for "scripts"
clic.txt is bad place for all of this

**Review by gigimon — APPROVED** [review]

**Comment by gigimon** [comment]
@IvanAdamovNeon looks good, please merge and deploy metrics script with devops to production

**Pull request merged**